code-challenges.js

```diff
@@ -17,33 +17,115 @@ const fruit4 = "kiwi"
+// use built-in method .length to get the full length of the newly assigned variable combinedNumbers containing the combination of myNumbers1 and myNumbers2 resulting to an output of 10 when console logged.
+
 const myNumbers1 = [3, 7, 0, 36, -9]
 const myNumbers2 = [8, -7, 42, 9, 13]
 
+const combinedNumbers = myNumbers1.concat(myNumbers2)
+
+// console.log(myNumbers1)
+// console.log(myNumbers2)
+
+// console.log(combinedNumbers.length)
 
 // --------------------3) Create the code that will reverse the letters of a string using the test variable provided below. Expected output: "1202 ohcE"
 
-const currentCohort = "Alpha 2022"
+// to come up with the expected output of "1201 ohcE", change "Alpha 2022" to "Echo 2021" - typo error correction
+// create a function to transform string into an array using .split to split the string "Echo 2021" into an array of letters
+// call built-in array method .reverse to reverse the order of the items in the array
+// join reveresed items in the array using .join to convert array into string
 
+// const currentCohort = "Echo 2021"
+
+// const transformToArray = currentCohort.split('')
+// const currentCohortReversed = transformToArray.reverse()
+// const joinReversed = currentCohortReversed.join('')
+// console.log(joinReversed)
 
 // --------------------4) Create a statement that will evaluate whether each number in an array is even or odd. Use the test variable provided below. Expected output: "odd" "even" "odd" "even" "odd" "even"
 
-const myArray = [13, 34, 5, 10, 27, 42]
 
+// use for loop to define where the count(index) starts; let i=0
+// define how many iterations the loop will go through; i < myArray.length
+// increment variable on each iteration
+// if statement is true using mod % operator resulting in 0, then return string 'even'
+// if result of mod % 2 is not 0, return 'odd'.
+
+// const myArray = [13, 34, 5, 10, 27, 42]
+
+// for (let i = 0; i < myArray.length; i++){
+//     if(myArray[i] % 2 === 0){
+//       console.log("even")
+//     } else {
+//       console.log("odd")
+//     }
+//   }
+ 
 
 // --------------------5) Create a function that takes in a set of any two numbers and subtracts the smaller number from the larger number using the sets of test variables provided below. Expected output: 42, 3
 
 // Set one:
-const number1 = 58
-const number2 = 100
+// const number1 = 58
+// const number2 = 100
+
+// create a function called subtraction that will take in test variables number1 and number2
+// use given test variables in the function
+// use "if statement" to determine the larger number that will return the smaller number to be deducted from the larger number
+
+
+// const subtraction = (number1, number2) => {
+//     if (number1 < number2){
+//       return (number2 - number1)
+//     } else {
+//       return (number1 - number2)
+//     }
+//   }
+//   console.log(subtraction(number1,number2))
```

Comment:
Since the test variables are `number1` and so on, you might consider creating a reusable function that takes in any two numbers. That way you only write one function that works for both sets of test variables.